_docs/setup.md

```diff
@@ -17,7 +17,7 @@ Clone any of your desired repositories and open them in your IDE.
 
 As an Administrator, you need to grant access to the desired repositories you want to use the GitLive plugin/extension with. This requires installing the GitLive app on your organization level or account level.
 
-Proceed to the [setup guide for administrators](/docs/admin) to install the GitLive app on your repository hosting service.
+If you are using a cloud repository hosting service (e.g. Github), proceed to the [setup guide for administrators (Cloud Repository Hosting Services)](/docs/admin) to install the GitLive app on your repository hosting service. If you are using a self-hosted repository (Currently GitLve only supports GitLab self-hosted), proceed to the [setup guide for administrators (GitLab Self-Hosted Repository)](/docs/admin-self-hosted). 
```

Comment:
there are more things on this page that are not accurate..

Reply:
also make this Github.com which stands for the cloud version

Reply:
install the GitLive app on your repository hosting service.  --> to grant permissions to the relevant repositories